src/components/app/StrategyProtocolList/index.tsx

```diff
@@ -93,29 +110,73 @@ type StrategyProtocolListProps = {
+    activation: number;
+    activationStr: string;
+    estimatedTotalAssetsUsdcNumber: number;
+    totalTvlPercentage: number;
+    debtOutstandingUsdcNumber: number;
+    dustUsdcNumber: number;
+    tvlImpact: number;
+};
+
 export const StrategyProtocolList = (props: StrategyProtocolListProps) => {
     const { network = DEFAULT_NETWORK } = useParams<ParamTypes>();
 
     const classes = useStyles();
-    const strategies = props.item.strategies.map((strategyTVL) => {
-        const amountInMMs = amountToMMs(strategyTVL.estimatedTotalAssetsUsdc);
-        return {
-            vault: strategyTVL.vault,
-            network,
-            strategy: strategyTVL.address,
-            name: strategyTVL.name,
-            activation: Date.parse(strategyTVL.params.activation),
-            activationStr: strategyTVL.params.activation,
-            estimatedTotalAssetsUsdcNumber: amountInMMs,
-            totalTvlPercentage: strategyTVL.estimatedTotalAssetsUsdc.isZero()
-                ? strategyTVL.estimatedTotalAssetsUsdc.toNumber()
-                : strategyTVL.estimatedTotalAssetsUsdc
-                      .times(100)
-                      .div(props.item.tvl)
-                      .toNumber(),
-            tvlImpact: getTvlImpact(amountInMMs),
-        };
-    });
+    const strategies: StrategyTVLListItem[] = props.item.strategies.map(
+        (strategyTVL) => {
+            const assetsInMMs = amountToMMs(
+                strategyTVL.estimatedTotalAssetsUsdc
+            );
+            const debtInMMs = amountToMMs(strategyTVL.debtOutstandingUsdc);
+            return {
+                vault: strategyTVL.vault,
+                network,
+                strategy: strategyTVL.address,
+                name: strategyTVL.name,
+                activation: Date.parse(strategyTVL.params.activation),
+                activationStr: strategyTVL.params.activation,
+                estimatedTotalAssetsUsdcNumber: assetsInMMs,
+                totalTvlPercentage:
+                    strategyTVL.estimatedTotalAssetsUsdc.isZero()
+                        ? strategyTVL.estimatedTotalAssetsUsdc.toNumber()
+                        : strategyTVL.estimatedTotalAssetsUsdc
+                              .times(100)
+                              .div(props.item.tvl)
+                              .toNumber(),
+                debtOutstandingUsdcNumber: debtInMMs,
+                dustUsdcNumber: strategyTVL.dustUsdc.toNumber(),
+                tvlImpact: getTvlImpact(assetsInMMs),
+            };
+        }
+    );
+
+    /**
+     * Returns true based on the following formula
+     * totalAssets > 2 * totalDebt && totalAssets > Dust
+     * @param item
+     */
+    const hasHealthyAssetsToDebtRatio = async (item: StrategyTVLListItem) => {
+        return (
+            item.estimatedTotalAssetsUsdcNumber >
+                item.debtOutstandingUsdcNumber * 2 &&
+            item.estimatedTotalAssetsUsdcNumber > item.dustUsdcNumber
```

Comment:
sorry maybe I misunderstood but based on the brief Telegram convo, I thought Dust would be hardcoded to something, so we wouldn't need `item.dustUsdcNumber`, and instead we could check if `item.estimatedTotalAssetsUsdcNumber > HARDCODED_DUST_VALUE`? eg) $1 USD for the dust threshold

Reply:
Yeah that might have been misleading since dust is essentially any leftover balance in the strategy's token, so to my understanding it itself is the threshold.
We could add a third check to make sure `dustUsdcNumber > DUST_THRESHOLD` with a loose-enough threshold of 100$ (?) 

@storming0x  @edwinzhng  lmk what you think here

Reply:
i think that works, we can test it and see if we get any reported issues

Reply:
I think `item.dustUsdcNumber` needs to be in MM.
`item.estimatedTotalAssetsUsdcNumber` is in MM, but `item.dustUsdcNumber` is not.
For exact comparison, `item.dustUsdcNumber` should be in MM.

Reply:
That's right, fixed on the latest commit, thanks for your comment!